tableauserverclient/models/user_item.py

```diff
@@ -98,6 +99,13 @@ def workbooks(self):
             raise UnpopulatedPropertyError(error)
         return self._workbooks()
 
+    @property
+    def favorites(self):
+        if self._favorites is None:
+            error = "User item must be populated with workbooks first."
```

Comment:
The string should say something like "favorites" or "content" since they can be more than just Workbooks

Reply:
Made the object name more generic.